google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1alpha2/StreamWriter.java

```diff
@@ -265,6 +273,28 @@ public Boolean expired() {
     return outstandingAppend.appendResult;
   }
 
+  /**
+   * This is the general flush method for asynchronise append operation. When you have outstanding
+   * append requests, calling flush will make sure all outstanding append requests completed and
+   * successful. Otherwise there will be an exception thrown.
+   *
+   * @throws Exception
+   */
+  public void flushAll() throws Exception {
```

Comment:
add a deadline here to avoid unexpected waiting time.

Reply:
Done.